src/IsolatedRidgeCorrectionFactor.cpp

```diff
@@ -16,25 +16,47 @@ double IsolatedRidgeCorrectionFactor(double d1_hzn__km, double d2_hzn__km, doubl
     double d_1__km[3] = { 15.0, 30.0, 60.0 };
     double d_2__km[9] = { 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 };
 
-    // points from Figure 31, Okumura
+    // points from Figure 31, Okumura, at corresponding d2 distances
     double curve_data[3][9] =
             { {  4.0, -13.0, -17.5, -17.5, -15.0, -12.5, -10.0, -8.0, -6.0 },   // C curve : d1 <= 15 km
               { 12.0,  -8.5, -13.0, -12.0, -10.0,  -8.0,  -6.5, -5.0, -4.0 },   // B curve : d1 <= 30 km
               { 20.0,  -4.0,  -6.5,  -6.0,  -4.5,  -3.5,  -2.5, -2.0, -1.0 } }; // A curve : d1 <= 60 km
 
     // normalized ridge height factor
-    double alpha = sqrt(h_edge__meter / 200.0); // Eq 1, Okumura, alpha = 0.07 * sqrt(h)
+    // Eq 1, Okumura, alpha = 0.07 * sqrt(h)
+    double alpha = sqrt(h_edge__meter / 200.0);
 
+    // identify the d1 curve set to use
     int id1 = 0;
     if (d1_hzn__km >= d_1__km[1])
         id1 = 1;
-    int id2 = 0;
 
+    // select the first d2 curve distance that is <= to actual path d2 distance
```

Comment:
Comment might be slightly confusing. Would it be more correct to say something like the following?
`select the first d2 curve distance that is directly below or equal to actual path d2 distance`

Reply:
the term "below" is problematic, as Figure 31 in the paper has plots the curves (from top to bottom) A,B,C, while the data array in the code contains the curve data as C,B,A, causing an ambiguity.  I'm going to leave the comment it as-is for now.